src/chainparams.cpp

```diff
@@ -53,7 +53,7 @@ class CMainParams : public CChainParams {
         pchMessageStart[2] = 0x4e;
         pchMessageStart[3] = 0xd8;
         vAlertPubKey = ParseHex("04fc9702847840aaf195de8442ebecedf5b095cdbb9bc716bda9110971b28a49e0ead8564ff0db22209e0374782c093bb899692d524e9d6a6956e7c5ecbcd68284");
-        nDefaultPort = 8333;
+        nDefaultPort = 8233;
```

Comment:
If you look at neighbouring ports, iTunes comes up too. So it's likely a range of ports that is randomly-selected from, so IMHO not significant at all.